conductor/submitter_nuke.py

```diff
@@ -253,8 +253,6 @@ def runPreSubmission(self):
         # save their script before continuing with submission
         if not self.checkSaveBeforeSubmission():
             raise exceptions.UserCanceledError()
-        else:
-            nuke_utils.save_current_nuke_script()
```

Comment:
I don't think you meant to remove the `nuke_utils.save_current_nuke_script()` line, but just the `else:` block, and dedent `nuke_utils.save_current_nuke_script()`.

Reply:
Okay, maybe I do need a coffee this morning... Sorry about that.